javascript/javascript3/redo/week3/shoppingCart.js

```diff
@@ -0,0 +1,82 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
+    switch (currency){
+      case "dollars": return this.price *= 0.16; break;
+      case "euro": return this.price *= 0.13; break;
+      case "pound": return this.price *= 0.11; break;
+  }
+  }
+}
+
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+  }
+
+  addProduct(product) {
+    this.products.push(product)
+  }
+
+  removeProduct(product) {
+    for (let i = 0; i < this.products.length; i++) {
+      if (this.products[i].name === product.name) {
+        this.products.splice(i, 1)
+      }
+    }
+  }
+
+  searchProduct(productName) {
+    const productArray = []
+    this.products.forEach(product => {
```

Comment:
We could have used the `filter` method here to return the array directly without having to declare `productArray`.